packages/front-end/components/Layout/Layout.tsx

```diff
@@ -27,9 +27,35 @@ const navlinks: SidebarLinkProps[] = [
     name: "Features",
     href: "/features",
     Icon: BsFlag,
-    path: /^features/,
-    beta: false,
-    className: styles.first,
+    path: /^(features|attributes|namespaces|environments|saved-groups)/,
+    autoClose: true,
+    subLinks: [
+      {
+        name: "All Features",
+        href: "/features",
+        path: /^features/,
+      },
+      {
+        name: "Attributes",
+        href: "/settings/attributes",
+        path: /^settings\/attributes/,
+      },
+      {
+        name: "Namespaces",
+        href: "/namespaces",
+        path: /^namespaces/,
+      },
+      {
+        name: "Environments",
+        href: "/settings/environments",
+        path: /^settings\/environments/,
```

Comment:
A lot of these pages currently require admin permissions to use.  To avoid people seeing errors, maybe we can hide Environments, Namespaces, and Attributes if the person doesn't have `organizationSettings` permission.  There's a `permissions` key you can define here that will do that automatically.